aidial_adapter_openai/utils/prompt_truncate.py

```diff
@@ -0,0 +1,65 @@
+from typing import Callable, List, Set, Tuple, TypeVar
```

Comment:
Why the inversion `prompt_truncate`? It would have made sense for `prompt_truncation`, but when you have a verb, the correct order is `truncate_prompt` - same as the method in the module and an endpoint in DIAL API.

Reply:
Fixed